tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Common/KeyPoint.java

```diff
@@ -63,6 +65,15 @@ public KeyPoint(String x, String y) throws RuntimeError {
         }
         pointer = result;
     }
+    
+    public KeyPoint(String address) throws RuntimeError {
+        RuntimeError error = new RuntimeError();
+        long result = jniKeyPointNewAddr(address, error);
+        if (error.code != 0) {
+            throw error;
+        }
+        pointer = result;
+    }
 
     /**
```

Comment:
Method is missing, `public String get_public_key(format: PublicKeyEncoding) throws RuntimeError`

this method will call key_point_encode(pointer, format, error)